integration/cli-login.expect

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env expect -f
+
+#
+# SUCCESS:
+# $ dcos auth login
+# dcos-host.com's username: good
+# good@dcos-host.com's password:
+# [core.dcos_acs_token]: set
+# Login successful!
+#
+# FAILURE:
+# $ dcos auth login
+# Removed [core.dcos_acs_token]
+# dcos-host.com's username: bad
+# bad@dcos-host.com's password:
+# dcos-host.com's username: bad
+# bad@dcos-host.com's password:
+# dcos-host.com's username: bad
+# bad@dcos-host.com's password:
+# Authentication failed
+#
```

Comment:
I see. Didn't realize that.

+1 for making a shared library. But, do you think we can upstream this common logic into Shakedown ? If the common logic is use-ful for C*, Kafka and HDFS, then its very likely that it will be useful for other frameworks as well. Thoughts ?